src/testUtils/java/org/broadinstitute/hellbender/testutils/VariantContextTestUtils.java

```diff
@@ -500,20 +521,24 @@ public static void assertVariantContextsHaveSameGenotypes(final VariantContext a
      *   is made about any other genotype fields which depend on the number of Alleles which might result in false negatives.
      * - This test requires that all attribute keys from the variant context are present, if one is writing a test and needs
      *   a complete header, consider {GATKVCFHeaderLine.getCompleteHeader()}
-     *
      * @param actual                Variant context to test for equality
      * @param expected              Expected result
-     * @param attributesToIgnore    Attributes we want to exclude from comparision
+     * @param attributesToIgnore    Attributes we want to exclude from numerical comparision, but ensure they exist
+     * @param attributesWithJitter
```

Comment:
I don't think the comment here is right -- your code above ensures that `attributesWithJitter` exist, but doesn't check the existence of the `attributesToIgnore`